trajectory_follower_wrapper/package.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+
+<!--
+ Copyright (C) 2024 LEIDOS.
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+-->
+
+<package format="3">
+  <name>trajectory_follower_wrapper</name>
+  <version>1.0.0</version>
+  <description>The trajectory_follower_wrapper package</description>
+
+  <maintainer email="carma@dot.gov">carma</maintainer>
+
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>carma_cmake_common</build_depend>
+  <build_depend>ament_auto_cmake</build_depend>
+
+  <depend>rclcpp</depend>
+  <depend>carma_ros2_utils</depend>
+  <depend>rclcpp_components</depend>
+  <depend>carma_guidance_plugins</depend>
+  <depend>autoware_msgs</depend>
+  <depend>autoware_config_msgs</depend>
```

Comment:
Looks like we can remove `autoware_config_msgs` here

Reply:
removed